standardize_mutation_data.py

```diff
@@ -949,7 +949,7 @@ def extract_vcf_format_info_data(vcf_data, tumor_sample_data_col, matched_normal
         # if 'FUNC' is a key in the 'INFO' data then promote key-value pairs to primary
         # key-value pairs in vcf_data dictionary - "FUNC" may contain important tumor or
         # normal sample data such as reads, alleles, etc.
-        if key.startswith("FUNC"):
+        if key=="FUNC":
```

Comment:
I'm on the fence as to whether we should accept this change or not, as it's been a few years since I've worked on this.

The use of  `.startswith()` instead of `==` may be due to [this](https://github.com/genome-nexus/annotation-tools/pull/46/files#diff-d297f3a61a8fd97249bc8b38d9214cffc1cc2d3a31441198616d6c793db51eaeR947) parsing of the VCF `INFO` data resulting in a string where the parsed `key` contains more than just `FUNC`.

The code below looks like it's just performing some additional processing on [value](https://github.com/genome-nexus/annotation-tools/pull/46/files#diff-d297f3a61a8fd97249bc8b38d9214cffc1cc2d3a31441198616d6c793db51eaeR944) only and not referring back to the parsed value of `key`. 

VCF data is a little fun(c)ky so my recommendation would be to revert the change here.

Reply:
@ruslan-forostianov what do you think? Was there a specific reason for updating this?

Reply:
@ao508 @sheridancbio What is `FUNC` field for? Does it have anything to do with GATK funcotator tool ?

Reply:
@ruslan-forostianov I have no memory of this.

I wonder if @thomasyu888 has any knowledge of the specificity of the key "FUNC" in the vcf INFO fields and whether narrowing the pattern match to exactly match "FUNC" rather than .startswith("FUNC") is safe? (context : collaborators who are leveraging this tool have seen a field "FUNCOTATOR" in their MAF files and the startswith() match was overgeneralizing in their data files.) I would guess that we are ok if the INFO field has a limited/standardized key vocabulary which does not contain any other keys which start with "FUNC..."

I will approve this PR now, but let's wait a little while before squashing/merging the PR (a week?) to see if there is any feedback on this point.

Reply:
I have a vague recollection that this was probably implemented to support some of the vcf formats we were receiving that:

1. Have information annotated via ANNOVAR. 
2. Only have the INFO column without the sample columns

I'd have to dig deeper within all the input vcfs to see why this was implemented.  I did notice that there were vcf's were `INFO=<ID=Func.ensGeneMT...`, but thats not capitalized.

Reply:
@thomasyu888 if you could do a scan of the input vcf's and reduce the set of recognized values to a specific list, maybe that would be the best option. If it does not seem feasible to do the scan, we could proceed in alternative ways, such as suggested above. (And let's hope one of the recognized values from your scan is not 'FUNCOTATOR' 😄 )